records_mover/records/targets/table/move_from_fileobjs_source.py

```diff
@@ -21,9 +24,9 @@ def __init__(self,
         self.fileobjs_source = fileobjs_source
         all_fileobjs = list(self.fileobjs_source.target_names_to_input_fileobjs.values())
         if len(all_fileobjs) != 1:
-            raise NotImplementedError("Teach me how to append and load more than one "
-                                      "file into same table")
-        self.fileobj = all_fileobjs[0]
+            self.fileobj = ConcatFiles(all_fileobjs)  # type: ignore
```

Comment:
I don't know how to convince the ConcatFiles class to type as `IO[bytes]`.  It certainly is duck-typed as one.